tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -44,3 +49,20 @@ def prepare_inputs_for_class(self, model_id=None):
         feature_extractor = AutoFeatureExtractor.from_pretrained("hf-internal-testing/tiny-random-ViTModel")
         inputs = feature_extractor(images=image, return_tensors="pt")
         return inputs
+
+
+class BetterTransformersViLTTest(BetterTransformersTestMixin, unittest.TestCase):
+    r"""
+    Testing suite for Vision and Text Models - tests all the tests defined in `BetterTransformersTestMixin`
+    """
+    all_models_to_test = ALL_VISION_TEXT_MODELS_TO_TEST
+
+    def prepare_inputs_for_class(self, model_id=None):
+        url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+        image = Image.open(requests.get(url, stream=True).raw)
+        text = "How many cats are there?"
+
+        # Model takes image and text as input
+        processor = AutoProcessor.from_pretrained("hf-internal-testing/tiny-random-ViltModel")
```

Comment:
```suggestion
        processor = AutoProcessor.from_pretrained(model_id)
```
We forgot to add this @ka00ri ! 💪 Now the tests should pass

Reply:
Good catch!  let's see🤞

Reply:
The current fail seems to be due to a missing file...I am not sure if I can fix this.